Run-SystemTest.ps1

```diff
@@ -0,0 +1,37 @@
+#Requires -Version 5.0
+
+Begin {
+    $ErrorActionPreference = "stop"
+}
+
+Process {
+    function Exec([scriptblock]$Command) {
+        & $Command
+        if ($LASTEXITCODE -ne 0) {
+            throw ("An error occurred while executing command: {0}" -f $Command)
+        }
+    }
+    
+    $workingDir = Join-Path $PSScriptRoot "src"
+    $outputDir = Join-Path $PSScriptRoot ".output"
+    $sysTestDir = Join-Path $PSScriptRoot "src/system-test/WebApi.MsBuild.SystemTest"
+
+    try {
+        Push-Location $workingDir
+        Remove-Item $outputDir -Force -Recurse -ErrorAction SilentlyContinue
+    
+        Exec { & dotnet pack -c Release -o "$outputDir" }
+
+        Push-Location $sysTestDir
+
+        Exec { & dotnet nuget locals all --clear }
+        Exec { & echo "<?xml version=`"1.0`" encoding=`"utf-8`"?><configuration><packageSources><add key=`"local-packages`" value=`"$outputDir`" /></packageSources></configuration>" > NuGet.Config }
+        Exec { & dotnet add package Workleap.OpenApi.MSBuild --prerelease }
+        Exec { & dotnet build -c Release }
```

Comment:
Do we want to assert that some things where done?

Reply:
Currently we make sure `$LASTEXITCODE` is 0 or else throw error. That should be enough for now, we can think about how to assert negative scenarios in future

Reply:
I feel that we don't know for sure that the OpenApi.MSBuild has run at all. If we remove all the code to add the reference it would still work.

Reply:
Would even be nice to have a system-test with an API not following the rules and validate that it break. (maybe an other card)